homeassistant/components/bsblan/climate.py

```diff
@@ -89,7 +90,7 @@ def __init__(
     ) -> None:
         """Initialize BSBLAN climate device."""
         super().__init__(client, device, info, static, entry)
-        CoordinatorEntity.__init__(self, coordinator)
+        super(CoordinatorEntity, self).__init__(coordinator)
```

Comment:
Both should work. However, there appears to be a bug within mypy for the former with the additional base class for `CoordinatorEntity`.
```
homeassistant/components/bsblan/climate.py:93: error: Argument 1 to "__init__" of "BaseCoordinatorEntity" has incompatible type "BSBLANClimate"; expected "BaseCoordinatorEntity[_BaseDataUpdateCoordinatorT]"  [arg-type]
homeassistant/components/bsblan/climate.py:93: error: Argument 2 to "__init__" of "BaseCoordinatorEntity" has incompatible type "DataUpdateCoordinator[Any]"; expected "_BaseDataUpdateCoordinatorT"  [arg-type]
```
Since the `super(...)` one worked, I used it here.

Reply:
lgtm